### PR TITLE
chore: add local `slither` script

### LINKED ISSUE #10: dx: support `slither` locally
[`slither`](https://github.com/crytic/slither) is python based.
Find a way to set it up smoothly in an way integrated with existing the node/yarn tooling.
Consider using a yarn `postinstall` hook that would run a `pip` install command?

### PR DESCRIPTION
**Pull request by sripwoud**
Fix #10

Add local yarn `slither` script.

### CODE REVIEW

File: .lintstagedrc.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "**/*.{js,ts,md,json,sol,yml,yaml}": "prettier --write"
+    "**/*.{js,ts,md,json,sol,yml,yaml}": "yarn prettier --write"
```
Comment (sripwoud) [inline .lintstagedrc.json:2]: am I the only one having an issue?
Locally lintstaged can't find the prettier bin otherwise

Comment (cedoor) [reply .lintstagedrc.json:2]: Ah, I may have prettier installed as a global package.. yarn prettier is better tho.

**Review by cedoor — APPROVED** [review]
utACK! Scripts are straightforward, I don't think I need to test them.

We should add more doc about these commands in the README files tho (?).

**Comment by sripwoud** [comment]
> We should add more doc about these commands in the README files tho (?).

Done https://github.com/privacy-scaling-explorations/zk-kit.solidity/pull/34/commits/adc30e79d50dc0cbbbcd5a30095ecf7f875f6c43

**Pull request merged**